Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -52,6 +75,17 @@ void main (void)
     
     vec3 rgb = rayleighPhase * v_rayleighColor + miePhase * v_mieColor;
     rgb = vec3(1.0) - exp(-exposure * rgb);
+    // compute luminance before color correction to avoid strangely gray night skies
     float l = czm_luminance(rgb);
+
+    // convert rgb color to hsv
+    vec3 hsv = rgb2hsb(rgb);
+    // perform hsv shift
+    hsv.x += u_hsvShift.x; // hue
+    hsv.y = clamp(hsv.y + u_hsvShift.y, 0.0, 1.0); // saturation
+    hsv.z = hsv.z > epsilon ? hsv.z + u_hsvShift.z : 0.0; // brightness
+    // convert shifted hsv back to rgb
+    rgb = hsb2rgb(hsv);
```

Comment:
I guess you'd have to compare performance, but if the user doesn't change these values (and I guess that includes the branch for dynamic atmosphere color as well), then it's doing unnecessary work. Maybe it's worth looking into ifdefs (like in `GlobeFS.glsl`).